src/easyscore.js

```diff
@@ -271,12 +292,18 @@ export class EasyScore {
     this.options = Object.assign({
       factory: null,
       builder: null,
+      commitHooks: [
+        setId,
+        setClass,
+        Articulation.easyScoreHook,
```

Comment:
I think the hook should be registered in `articulation.js`, not here. Ideally there will be no articulation code here, so new elements don't ever have to touch this file. You can implement this with an `EasyScore` static method. So the end of `articulation.js` will look like this:

```javascript
EasyScore.registerDefaultCommitHook(Articulation.easyScoreHook);
```